tests/tests/test_sfn.py

```diff
@@ -7,8 +7,11 @@
 import boto3
 
 LOCALSTACK_URI = "http://localstack:4566"
+
+WORKSPACE = os.popen("terraform workspace show").read().strip()
```

Comment:
nifty. We could also probably just pass the SFN ARN as an environment parameter from terraform, but this works fine

Reply:
Do you need the ARN in production, too? And do you build the SFN using a test terraform module or the orginal TF module?